components/core/src/clp/ffi/StringBlob.hpp

```diff
@@ -0,0 +1,66 @@
+#ifndef CLP_FFI_STRINGBLOB_HPP
+#define CLP_FFI_STRINGBLOB_HPP
+
+#include <cstddef>
+#include <optional>
+#include <string>
+#include <string_view>
+#include <vector>
+
+#include "../ErrorCode.hpp"
+#include "../ReaderInterface.hpp"
+
+namespace clp::ffi {
+// Stores a blob of string data with indexing capabilities.
```

Comment:
```suggestion
// Stores a list of strings as an indexable blob.
```
It's a little bit more clear this way I think.